extensions/iceberg/src/main/java/io/deephaven/iceberg/layout/IcebergBaseLayout.java

```diff
@@ -79,8 +96,8 @@ protected IcebergTableLocationKey locationKey(
             @Nullable final Map<String, Comparable<?>> partitions) {
         final org.apache.iceberg.FileFormat format = dataFile.format();
         if (format == org.apache.iceberg.FileFormat.PARQUET) {
-            return new IcebergTableParquetLocationKey(manifestFile, dataFile, fileUri, 0, partitions,
-                    parquetInstructions, channelsProvider);
+            return new IcebergTableParquetLocationKey(tableUuid, catalogName, tableIdentifier, manifestFile, dataFile,
+                    fileUri, 0, partitions, parquetInstructions, channelsProvider);
```

Comment:
That sure is a lot of constructor args. Sorry. Builder? (Probably not worth it for something called in only once place.)